browser/app.css

```diff
@@ -423,7 +423,7 @@ footer {
   }
 
   .back-btn-spacer {
-    right: 183px;
+    right: 200px;
```

Comment:
this will make space between 'Back' and 'Next' 13px wider than between 'Back' and 'Download and Install'.